src/hc-06.hpp

```diff
@@ -0,0 +1,99 @@
+// ==========================================================================
+//
+// File      : hc-06.hpp
+// Part of   : C++ library for using the hc-06 bluetooth module.
+//
+// ==========================================================================
+
+// this file contains Doxygen lines
+/**
+ * @class HC06
+ * @author HU R2D2 2018
+ * @date 13/05/2018
+ * @file hc-06.hpp
+ */
+
+
+#ifndef HC06_H
+#define HC06_H
+
+#include "wrap-hwlib.hpp"
+#include "bluetooth.hpp"
+
+/// \brief
+/// HC-06 bluetooth library
+class HC06 : public bluetooth{
+private:
+    uint8_t name[32]; /// Used for storing the name of this device.
+    uint8_t discoveredDevices[32]; /// Used for storing the connection id of a discovered device.
+public:
+    HC06();
+
+    /// \brief
+    /// Used to connect to other devices
+    /// \details
+    /// Connect to the specified device ID. To get the device ID use function "search".
+    /// @param[in] device ID
+    void connect(int deviceID);
+
+    /// \brief
+    /// Used to disconnect form a device.
+    void disconnect();
+
+    /// \brief
+    /// Used to read the name of this device.
+    /// @return current device name
+    uint8_t* getName();
+
+    /// \brief
```

Comment:
Incorrect use of Doxygen, please read the wiki for the correct Doxygen standard.